_projects/geotrellis-vt-server.md

```diff
@@ -0,0 +1,57 @@
+---
+# Remove the project attributes which don't apply to your project. (ie: if no issues, delete the issues section)
+
+published: false # change to true once ready to post
+
+title: "Dynamic Vector Tile Server" # project title inside quotes
+excerpt: "Serve dynamic vector tiles from GeoJSON and other sources using GeoTrellis" # shows on project list page
+
+# project attributes
+
+requirements: # bullet list of requirements – one requirement per line – follow below format
+  - "experience building or using APIs"
+  - "interest in Scala / FP"
+
+tags: # one tag per line – spaces are allowed in tags – use tags other posts use
+  - "vector tiles"
+  - "scala"
+  - "GeoTrellis"
+
+difficulty: "medium" # Easy, Medium, Hard – pick one
+
+mentors: # github username without the @ – example: designmatty
+  - "jisantuc"
+  - "cloudniner"
+  
+---
+
+# Motivation
+
+Prior art exists showing efficient vector tile slicing and rendering for static data. Raster Foundry and other projects have a need for the ability to serve dynamically generated vector tiles from data provided by users as they use the application. Several other options (t-rex, tegola, tilegarden) require configuration to serve a set of layers when the server starts, but the goal of this project is for clients to ad hoc specify layers.
+
+# Milestones
+
+## Preparation
+
+You will prepare by gaining a working knowledge of:
+
+  - scala development
+  - scala libraries for web servers
```

Comment:
list libraries you expect to be used, `http4s` could be a motivation or more information.

Reply:
Ah, it happens later, deleting.